console application exception with XBEE stick

After upgrading to com.zsmartsystems.zigbee.console.main-1.3.6-jar-with-dependencies.jar
got:

ZigBee network state updated to INITIALISING
Exception in thread "main" com.thoughtworks.xstream.InitializationException: Could not instantiate mapper : com.thoughtworks.xstream.mapper.LambdaMapper
	at com.thoughtworks.xstream.XStream.buildMapperDynamically(XStream.java:645)
	at com.thoughtworks.xstream.XStream.buildMapper(XStream.java:622)
	at com.thoughtworks.xstream.XStream.<init>(XStream.java:586)
	at com.thoughtworks.xstream.XStream.<init>(XStream.java:514)
	at com.thoughtworks.xstream.XStream.<init>(XStream.java:483)
	at com.thoughtworks.xstream.XStream.<init>(XStream.java:429)
	at com.thoughtworks.xstream.XStream.<init>(XStream.java:396)
	at com.zsmartsystems.zigbee.console.main.ZigBeeDataStore.openStream(ZigBeeDataStore.java:65)
	at com.zsmartsystems.zigbee.console.main.ZigBeeDataStore.readNode(ZigBeeDataStore.java:118)
	at com.zsmartsystems.zigbee.database.ZigBeeNetworkDatabaseManager.startup(ZigBeeNetworkDatabaseManager.java:193)
	at com.zsmartsystems.zigbee.ZigBeeNetworkManager.initialize(ZigBeeNetworkManager.java:379)
	at com.zsmartsystems.zigbee.console.main.ZigBeeConsoleMain.main(ZigBeeConsoleMain.java:287)
Caused by: java.lang.ClassNotFoundException: com.thoughtworks.xstream.mapper.LambdaMapper
	at com.thoughtworks.xstream.core.util.CompositeClassLoader.loadClass(CompositeClassLoader.java:148)
	at java.lang.Class.forName0(Native Method)
	at java.lang.Class.forName(Class.java:348)
	at com.thoughtworks.xstream.XStream.buildMapperDynamically(XStream.java:641)
	... 11 more

--- a/com.zsmartsystems.zigbee.console.main/pom.xml
+++ b/com.zsmartsystems.zigbee.console.main/pom.xml
@@ -83,7 +83,7 @@
 		<dependency>
 			<groupId>com.thoughtworks.xstream</groupId>
 			<artifactId>xstream</artifactId>
-			<version>1.4.10-java7</version>
+			<version>1.4.10</version>
 		</dependency>
 
 	</dependencies>

--- a/com.zsmartsystems.zigbee.console.main/src/main/java/com/zsmartsystems/zigbee/console/main/ZigBeeDataStore.java
+++ b/com.zsmartsystems.zigbee.console.main/src/main/java/com/zsmartsystems/zigbee/console/main/ZigBeeDataStore.java
@@ -12,7 +12,6 @@ import java.io.BufferedWriter;
 import java.io.File;
 import java.io.FileInputStream;
 import java.io.FileOutputStream;
-import java.io.IOException;
 import java.io.InputStreamReader;
 import java.io.OutputStreamWriter;
 import java.util.HashSet;
@@ -62,25 +61,30 @@ public class ZigBeeDataStore implements ZigBeeNetworkDataStore {
     }
 
     private XStream openStream() {
-        XStream stream = new XStream(new StaxDriver());
-        stream.alias("ZigBeeKey", ZigBeeKey.class);
-        stream.alias("ZigBeeNode", ZigBeeNodeDao.class);
-        stream.alias("ZigBeeEndpoint", ZigBeeEndpointDao.class);
-        stream.alias("ZclCluster", ZclClusterDao.class);
-        stream.alias("ZclAttribute", ZclAttributeDao.class);
-        stream.alias("MacCapabilitiesType", MacCapabilitiesType.class);
-        stream.alias("ServerCapabilitiesType", ServerCapabilitiesType.class);
-        stream.alias("PowerSourceType", PowerSourceType.class);
-        stream.alias("FrequencyBandType", FrequencyBandType.class);
-        stream.alias("BindingTable", BindingTable.class);
-        stream.alias("IeeeAddress", IeeeAddress.class);
-        stream.registerConverter(new IeeeAddressConverter());
+        try {
+            XStream stream = new XStream(new StaxDriver());
+            stream.alias("ZigBeeKey", ZigBeeKey.class);
+            stream.alias("ZigBeeNode", ZigBeeNodeDao.class);
+            stream.alias("ZigBeeEndpoint", ZigBeeEndpointDao.class);
+            stream.alias("ZclCluster", ZclClusterDao.class);
+            stream.alias("ZclAttribute", ZclAttributeDao.class);
+            stream.alias("MacCapabilitiesType", MacCapabilitiesType.class);
+            stream.alias("ServerCapabilitiesType", ServerCapabilitiesType.class);
+            stream.alias("PowerSourceType", PowerSourceType.class);
+            stream.alias("FrequencyBandType", FrequencyBandType.class);
+            stream.alias("BindingTable", BindingTable.class);
+            stream.alias("IeeeAddress", IeeeAddress.class);
+            stream.registerConverter(new IeeeAddressConverter());
 
-        // stream.registerLocalConverter(ZigBeeKey.class, "key", new KeyArrayConverter());
-        // stream.registerLocalConverter(ZigBeeKey.class, "address", new IeeeAddressConverter());
-        // stream.registerLocalConverter(BindingTable.class, "srcAddr", new IeeeAddressConverter());
-        // stream.registerLocalConverter(BindingTable.class, "dstAddr", new IeeeAddressConverter());
-        return stream;
+            // stream.registerLocalConverter(ZigBeeKey.class, "key", new KeyArrayConverter());
+            // stream.registerLocalConverter(ZigBeeKey.class, "address", new IeeeAddressConverter());
+            // stream.registerLocalConverter(BindingTable.class, "srcAddr", new IeeeAddressConverter());
+            // stream.registerLocalConverter(BindingTable.class, "dstAddr", new IeeeAddressConverter());
+            return stream;
+        } catch (Exception e) {
+            logger.debug("Error opening XStream ", e);
+            return null;
+        }
     }
 
     private File getFile(IeeeAddress address) {
@@ -122,7 +126,7 @@ public class ZigBeeDataStore implements ZigBeeNetworkDataStore {
         try (BufferedReader reader = new BufferedReader(new InputStreamReader(new FileInputStream(file), "UTF-8"))) {
             node = (ZigBeeNodeDao) stream.fromXML(reader);
             logger.info("{}: ZigBee reading network state complete.", address);
-        } catch (IOException e) {
+        } catch (Exception e) {
             logger.error("{}: Error reading network state: ", address, e);
         }
 

